packages/storage/src/indexed-db/indexed-db.test.ts

```diff
@@ -94,16 +94,14 @@ describe('IndexedDb', () => {
       const version1Props = generateInitialProps();
       const dbA = await IndexedDb.initialize(version1Props);
       await dbA.saveAssetsMetadata(metadataA);
+      dbA.close();
```

Comment:
The thing is that in order to upgrade idb to a new version you have to wait for it to close. 
If idb is not closed, the test will last forever.  
This did not lead to a bug only because when updating the version extension restarts and closes indexed-db